### ISSUE TITLE
Upgrade Spring Boot to the latest version

### ISSUE THREAD

**Issue by robertmcnees** [spring-guides/gs-messaging-rabbitmq#37]
Update the guide to use the most recent Spring Boot version.

Files that require changes are:
```
initial/build.gradle
initial/pom.xml
complete/build.gradle
complete/pom.xml
```

**Comment by EvandroMaddes** [comment]
@robertmcnees can you approve the pull request?

**Comment by robertmcnees** [comment]
Closed with #43
